Bindings/Python/__init__.py

```diff
@@ -1,15 +1,19 @@
 import sys
 import os
-if (sys.platform.startswith('win')):
-      curFolder = os.path.dirname(os.path.realpath(__file__))
-      os.add_dll_directory(curFolder)
-      # in install env, add relative path from init.py to bin, in build environment, we also add "../../../../Release"
-      install_path = os.path.join(curFolder, "../../../bin")
-      if (os.path.exists(install_path)):
-          os.add_dll_directory(install_path)
-      dev_path = os.path.join(curFolder, "../../../../Release")
-      if (os.path.exists(dev_path)):
-          os.add_dll_directory(dev_path)
+
+if sys.platform.startswith('win'):
+    curFolder = os.path.dirname(os.path.realpath(__file__))
+    os.add_dll_directory(curFolder)
+    # in install env, add relative path from init.py to bin, in build environment, we also add "../../../../Release"
+    try: # pass only if DLL_PATH is updated by setup.py in local installation
+        bin_path = DLL_PATH
```

Comment:
Not sure what DLL_PATH would be set to at this line.

Reply:
The `DLL_PATH` would be updated by `setup.py` to be the OpenSim bin path.